### PR TITLE
impl visual model

### PR DESCRIPTION
**Pull request by teamwong111**
## Motivation

Add model visualization in TensorboardLoggerHook.

## Modification

`mmcv/runner/hooks/logger/tensorboard.py` and `tests/test_runner/test_hooks.py` for model visualization.
`mmcv/runner/hooks/logger/pavi.py` for fixing docstring by the way.

## BC-breaking (Optional)

- Downstream repos need to modify a few codes of `forward_test`, like [mmcls](https://github.com/open-mmlab/mmclassification/blob/master/mmcls/models/classifiers/base.py#L69). Because although `add_graph` can pass in `List[Tensor]`, `torch/jit/_trace.py` has a loop and the parameter into `Module.forward` is a `tuple(Tensor)`.

## Checklist

**Before PR**:

- [x] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [x] If the modification has potential influence on downstream or other related projects, this PR should be tested with some of those projects, like MMDet or MMCls.
- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: mmcv/runner/hooks/logger/tensorboard.py [hunk 1]
```diff
@@ -1,15 +1,20 @@
 # Copyright (c) OpenMMLab. All rights reserved.
 import os.path as osp
+from functools import partial
 
-from mmcv.utils import TORCH_VERSION, digit_version
+import torch
+
+from ....parallel import DataContainer
+from ....parallel.utils import is_module_wrapper
+from ....utils import TORCH_VERSION, digit_version
```
Comment (mzr1996) [inline mmcv/runner/hooks/logger/tensorboard.py:9]: Relative package path with more than two `.` is weird. Why don't use absolte path?

File: tests/test_runner/test_hooks.py [hunk 2]
```diff
@@ -21,12 +21,15 @@
 from torch.utils.data import DataLoader
 
 from mmcv.fileio.file_client import PetrelBackend
+# yapf: disable
```
Comment (mzr1996) [inline tests/test_runner/test_hooks.py:24]: Why disable yapf here?

**Review by mzr1996 — CHANGES_REQUESTED** [review]

File: mmcv/runner/hooks/logger/tensorboard.py (hunk 1 above)
Comment (teamwong111) [reply mmcv/runner/hooks/logger/tensorboard.py:9]: I will fix it.

File: tests/test_runner/test_hooks.py (hunk 2 above)
Comment (teamwong111) [reply tests/test_runner/test_hooks.py:24]: Because isort and yapf are conflict.

**Pull request closed without merging**